packages/core/src/odata/v4/filter/filter-lambda-expression.ts

```diff
@@ -0,0 +1,38 @@
+/* Copyright (c) 2020 SAP SE or an SAP affiliate company. All rights reserved. */
+import { FieldType } from '../../common/selectable';
+import { Filterable, FilterList } from '../../common/filter';
+import { EntityBase } from '../../common';
+import { OneToManyLink } from '../../common/selectable/one-to-many-link';
+import { FilterLambdaExpression } from '../../common/filter/filter-lambda-expression';
+// eslint-disable-next-line valid-jsdoc
+/**
+ * @experimental This is experimental and is subject to change. Use with caution.
+ */
+export function any<FieldT extends FieldType, EntityT extends EntityBase>(
+  ...filters: (Filterable<EntityT> | OneToManyLink<EntityT>)[]
```

Comment:
I can't think of an example where you would pass a one to many link. Would't that mean something like the example below? If yes, that should not be possible.

```
TestEntity.requestBuilder().getAll()
  .filter(any(TestEntity.to_MultiLink))
```

Reply:
I created a backlog item for renaming the `filter()`